README.md

```diff
@@ -4,24 +4,24 @@ Python courses for the scientific researcher
 This repository contains several courses for the benefit of scientific researchers,
 particularly in the fields of oceanography and meteorology.
 
-There are currently five courses:
+There are currently 5 courses:
```

Comment:
Ooh, could you change this back – numbers less than 10 should be rendered as words.

Reply:
I really have awoken the beast ...

Reply:
🐻